packages/common/src/eips.ts

```diff
@@ -190,6 +191,23 @@ export const EIPs: EIPsDict = {
       },
     },
   },
+  2935: {
+    comment: 'Save historical block hashes in state',
```

Comment:
Can we add some addition like `...state (Verkle related usage, likely not stable yet)` or so?

(let me know if the comment does not make sense "as is")

Reply:
👍 

Take whatever text you like, just that people know this is not "one of the normal EIPs they can integrate" (and DRAFT status is not *that* much of a warning, even EIPs close to hardfork often still in DRAFT 😂 ).